Tools/Scripts/webkitpy/w3c/test_importer.py

```diff
@@ -121,8 +121,13 @@ def format(self, record):
 # FIXME: We should decide whether we want to make this specific to web-platform-tests or to make it generic to any git repository containing tests.
 def parse_args(args):
     description = """
-To import a web-platform-tests test suite named xyz, use: 'import-w3c-tests web-platform-tests/xyz'.
-To import a web-platform-tests suite from a specific folder, use 'import-w3c-tests xyz -s my-folder-containing-web-platform-tests-folder'"""
+To import a web-platform-tests test suite named xyz, use:
+    import-w3c-tests --tip-of-tree web-platform-tests/xyz
+
+To import a web-platform-tests suite from a specific local checkout out of web platform tests:
+   1. Your checkout must be a folder called "web-platform-tests".
+   2. If the path is "~/dev/web-platform-tests/", use:
```

Comment:
```suggestion
   2. If the working tree is at "~/dev/web-platform-tests", use:
```

Reply:
```suggestion
   2. If the local copy is at "~/dev/web-platform-tests/", use:
```

Reply:
I would say "If the local copy is at, for example, ..."

Reply:
done.